apluslms_shepherd/celery_tasks/build/observer.py

```diff
@@ -0,0 +1,107 @@
+import sys
+from datetime import datetime
+from enum import Enum
+
+from apluslms_roman.observer import BuildObserver
+
+from apluslms_shepherd.build.models import BuildAction, BuildState, Build, BuildLog
+from apluslms_shepherd.extensions import celery, db
+
+
+class Message(Enum):
```

Comment:
Why this is copied here and not just imported?

Reply:
I add a new item in here`STATE_UPDATE = 13`, but yeah, I should inherit rather than copy